### ISSUE TITLE
'sphinx/writers/latex.py' missing in_production_list check

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1619]
Similar to #1278, "sphinx/writers/latex.py" contains a second reference to latex_show_pagerefs but it is not guarded by an in_production_list check.  I'm getting documents that have pagerefs inside production lists.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1619
- Originally reported by: Brad King
- Originally created at: 2014-11-06T20:24:48.696

**Comment by shimizukawa** [comment]
_From Brad King on 2014-11-06 19:28:16+00:00_

This fixes it for me:

```
diff -r 8fddbe9f9805 sphinx/writers/latex.py
--- a/sphinx/writers/latex.py   Thu Nov 06 09:12:51 2014 +0100
+++ b/sphinx/writers/latex.py   Thu Nov 06 14:24:31 2014 -0500
@@ -1236,7 +1236,8 @@
                 # don't add a pageref for glossary terms
                 self.context.append('}}')
             else:
-                if self.builder.config.latex_show_pagerefs:
+                if self.builder.config.latex_show_pagerefs and not \
+                    self.in_production_list:
                     self.context.append('}} (%s)' % self.hyperpageref(id))
                 else:
                     self.context.append('}}')
```

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-11-06 20:26:23+00:00_

Closes #1619: missing in_production_list check for latex writer

→ <<cset 5d9d80227239047764271eb9f90c4ae4715a0b24>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-11-06 20:26:35+00:00_

Thanks!
